meta-client/src/main/java/org/apache/helix/metaclient/exception/MetaClientException.java

```diff
@@ -35,4 +35,114 @@ public MetaClientException(String message) {
   public MetaClientException(Throwable cause) {
     super(cause);
   }
+
+  public enum Code {
+    /** Everything is OK. */
+    OK,
+
+    /** Indicates a system and server-side errors not defined by following codes.
+     * It also indicate a range. Any value larger than this and less than DBUSERERROR
+     * indicating error from server side.*/
+    DBSYSTEMERROR,
+
+    /** Connection to the server has been lost. */
+    CONNECTIONLOSS,
+
+    /** Operation is unimplemented. */
+    UNIMPLEMENTED,
+
+    /** Operation timeout. */
+    OPERATIONTIMEOUT,
+
+    /** Either a runetime or data inconsistency was found. */
+    CONSISTENCYERROR,
```

Comment:
Same here. Better to have one word and use underscore connecting them. This is Helix tradition. I know ZK may have a different style. But let's follow how Helix defines.